internal/ls/utilities.go

```diff
@@ -268,10 +268,7 @@ func (l *LanguageService) createLspRangeFromNode(node *ast.Node, file *ast.Sourc
 }
 
 func (l *LanguageService) createLspRangeFromBounds(start, end int, file *ast.SourceFile) *lsproto.Range {
-	lspRange, err := l.converters.ToLSPRange(file.FileName(), core.NewTextRange(start, end))
-	if err != nil {
-		panic(err)
-	}
+	lspRange := l.converters.ToLSPRange(file, core.NewTextRange(start, end))
 	return &lspRange
```

Comment:
Not related to anything you're doing here but I think it's weird that we have this pointer to range when most everyone actually just dereferences it immediately. (I think it's used in one place where it's optional but it seems to me like the pointer-ness should be moved down there.)

Reply:
I'll leave it alone in this PR. cc @gabritto if she wants to consider changing it.